cmd/fleetctl/package.go

```diff
@@ -336,6 +342,20 @@ func packageCommand() *cli.Command {
 				return errors.New("--use-system-configuration is only available for pkg installers")
 			}
 
+			linuxPackage := false
+			switch c.String("type") {
+			case "msi", "deb", "rpm":
```

Comment:
Is `msi` really supposed to be there? If so the `Usage` text should probably be adjusted, but I think msi is Windows-only?

Reply:
Nice catch! That isn't supposed to be there